kubernetes/crds/machine.sapcloud.io_machineclasses.yaml

```diff
@@ -55,6 +55,9 @@ spec:
             description: NodeTemplate contains subfields to track all node resources
               and other node info required to scale nodegroup from zero
             properties:
+              architecture:
+                description: Architecture of the node belonging to nodeGroup
```

Comment:
This is actually a processor/CPU architecture. Should we call it that?

Reply:
The field is called architecture by convention in k8s. Example [NodeSystemInfo.Architecture](https://pkg.go.dev/k8s.io/api/core/v1#NodeSystemInfo.Architecture)

Reply:
Maybe i was not clear. I was not referring to the name of the property which should be called `Architecture` and that is the reason my comment is for the `description` line where it refers to architecture as `Architecture of the node`

Reply:
Sure will make the changes.